client/lib/explat/internals/fetch-experiment-assignment.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * WordPress dependencies
+ */
+import wpcom from 'calypso/lib/wp';
+
+// Using typescript to ensure we are being safe in SSR
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+declare const window: undefined | ( Window & typeof globalThis );
+
+export default function fetchExperimentAssignment( {
+	experimentName,
+	anonId,
+}: {
+	experimentName: string;
+	anonId: string | null;
+} ): Promise< unknown > {
+	return wpcom.req.get(
+		{
+			path: '/experiments/0.1.0/assignments/wpcom',
```

Comment:
Can you please add an issue to remind us to change this to `/calypso`?

Reply:
Created: 579-gh-Automattic/experimentation-platform